wire/internal/wire/parse.go

```diff
@@ -191,9 +191,9 @@ type Value struct {
 	info *types.Info
 }
 
-// Load finds all the provider sets in the given packages, as well as
-// the provider sets' transitive dependencies. It may return both errors
-// and Info.
+// Load finds all the provider sets in the packages that match the given
```

Comment:
Can you define "match"? Is pattern a regexp, or is it a partial match, or a full string match, or ...?

Reply:
There is currently no agreed upon definition of this pattern: it is (will be) at the whims of what [`go/packages`](https://godoc.org/golang.org/x/tools/go/packages#pkg-overview) documents. How do you wish to proceed?

Reply:
I would suggest just saying that. E.g., "... that match the given patterns according to the match rules defined by https://godoc.org/golang.org/x/tools/go/packages#pkg-overview..."

Reply:
Done.